app/src/lib/stores/github-user-store.ts

```diff
@@ -158,6 +183,26 @@ export class GitHubUserStore extends BaseStore {
       )
       .map(h => h.user)
   }
+
+  private setQueryCache(
+    repository: GitHubRepository,
+    users: ReadonlyArray<IMentionableUser>
+  ) {
+    this.clearCachePruneTimeout()
+    this.queryCache = { repository, users }
+    // Clear mentionables cache after one minute
+    this.pruneQueryCacheTimeoutId = window.setTimeout(() => {
+      this.pruneQueryCacheTimeoutId = null
+      this.queryCache = null
+    }, 60 * 1000)
```

Comment:
Let's put this value in a constant to be consistent with the other expiration timeouts

Reply:
https://github.com/desktop/desktop/pull/10121/commits/4c31714e1fefaba6e7e917fc5ecfc83461114889